source/renderer/app/components/widgets/forms/WalletsDropdown.js

```diff
@@ -33,37 +35,50 @@ export const onSearchWalletsDropdown = (
   });
 };
 
-export default class WalletsDropdown extends Component<Props> {
-  static defaultProps = {
-    onSearch: onSearchWalletsDropdown,
+export default function WalletsDropdown({
+  className,
+  getStakePoolById,
+  numberOfStakePools,
+  onSearch = onSearchWalletsDropdown,
+  wallets = [],
+  ...props
+}: Props) {
+  const discreetModeFeature = useDiscreetModeFeature();
+  const itemsDropdownProps = {
+    ...omit(props, ['wallets', 'options']),
+    onSearch,
   };
-  render() {
```

Comment:
Please avoid reformatting large parts as part of a feature PR as this makes it much harder to see what actually changed.

Reply:
Ah ok, if the hooks refactoring was necessary no worries 👍